src/crimson/osd/objclass.cc

```diff
@@ -356,7 +356,11 @@ int cls_cxx_map_clear(cls_method_context_t hctx)
 
 int cls_cxx_map_write_header(cls_method_context_t hctx, bufferlist *inbl)
 {
-  return 0;
+  OSDOp op{ CEPH_OSD_OP_OMAPSETHEADER };
```

Comment:
```suggestion
  OSDOp op{CEPH_OSD_OP_OMAPSETHEADER};
```

This recalls me we need to unify the spacing across the `objclass.cc`. Many functions there doesn't use the spaces (e.g. `OSDOp op{CEPH_OSD_OP_OMAPSETHEADER}`), so maybe avoid the extra spacing here?